docs/architecture/testing/driver-interface-test-strategy.md

```diff
@@ -384,8 +384,10 @@ The following entry points are declared (transparent and opaque):
 The transparent driver fully implements the declared entry points, and can use
 any backend: internal or libtestdriver1.
 
-The opaque driver is not implemented at all, neither instumentation nor the
-operation: entry points always return `NOT_SUPPORTED`.
+The opaque driver implements the declared entry points, and can use any
+backend: internal or libtestdriver1. However it does not implement the
+instrumentation (hits, forced output/status), as this [was not an immediate
```

Comment:
From a chat with Ronald: hits is actually probably an obsolete concern. With the internal backend, hits were how we knew that the driver was called, as opposed to directly calling the built-in code. With libtestdriver1, we check that by ensuring that the built-in code is not present, so if the operation gives the correct result, only a driver call can have calculated that result. So there is low value in checking the hit count. There is still some value for hit counts, e.g. checking that we don't call a multipart entry point when we intended to call the one-shot entry point, but it's limited.